cli/src/command/account.rs

```diff
@@ -680,20 +688,45 @@ pub async fn sync_command(account_handle: &AccountHandle) -> Result<(), Error> {
 }
 
 /// `transactions` command
-pub async fn transactions_command(account_handle: &AccountHandle) -> Result<(), Error> {
-    let transactions = account_handle.transactions().await?;
+pub async fn transactions_command(account_handle: &AccountHandle, show_details: bool) -> Result<(), Error> {
+    let mut transactions = account_handle.transactions().await?;
+    transactions.sort_by(|a, b| a.timestamp.cmp(&b.timestamp));
 
     if transactions.is_empty() {
         println_log_info!("No transactions found");
     } else {
-        for tx in transactions {
-            println_log_info!("{}", serde_json::to_string(&TransactionDto::from(&tx))?);
+        for (i, tx) in transactions.into_iter().enumerate() {
+            if show_details {
+                println_log_info!("{:#?}", tx);
+            } else {
+                let transaction_time = to_utc_date_time(tx.timestamp)?;
+                let formatted_time = transaction_time.format("%Y-%m-%d %H:%M:%S").to_string();
+
+                println_log_info!("{:<5} {:40}\t{:20}", i, tx.transaction_id.to_string(), formatted_time);
+            }
         }
     }
 
     Ok(())
 }
 
+/// `transaction` command
+pub async fn transaction_command(account_handle: &AccountHandle, transaction_id: &str) -> Result<(), Error> {
+    let maybe_transaction = account_handle
+        .transactions()
+        .await?
+        .into_iter()
+        .find(|tx| tx.transaction_id.to_string() == transaction_id);
```

Comment:
I think I'd rather convert `transaction_id` to a `TransactionId` once instead of converting to string in the loop. It would also actually validate the user input.

Reply:
good point, I was thingk bout that, but forgot to change. Fixed